formats/organisation.jsonnet

```diff
@@ -0,0 +1,571 @@
+          description: "Whether to prioritise news or services on the organisation's home page.",
+        },
+        organisation_govuk_status: {
+          type: "object",
+          additionalProperties: false,
+          required: [
+            "status",
+          ],
+          properties: {
+            status: {
+              type: "string",
+              enum: [
+                "changed_name",
+                "devolved",
+                "exempt",
+                "joining",
+                "left_gov",
+                "live",
+                "merged",
+                "no_longer_exists",
+                "replaced",
+                "split",
+                "superseded_by_devolved_administration",
+                "transitioning",
+              ],
+            },
+            updated_at: {
+              type: "string",
+              format: "date-time",
+            },
+          },
+          description: "The status of the organisation on GOV.UK.",
+        },
+        organisation_type: {
+          type: "string",
+          enum: [
+            "adhoc_advisory_group",
+            "advisory_ndpb",
+            "civil_service",
+            "court",
+            "devolved_administration",
+            "executive_agency",
+            "executive_ndpb",
+            "executive_office",
+            "independent_monitoring_body",
+            "ministerial_department",
+            "non_ministerial_department",
+            "other",
+            "public_corporation",
+            "sub_organisation",
+            "tribunal_ndpb",
+          ],
+          description: "The type of organisation.",
+        },
+        social_media_links: {
+          type: "array",
+          items: {
+            type: "object",
+            additionalProperties: false,
+            required: [
+              "service_type",
+              "title",
+              "href",
+              "image",
+            ],
+            properties: {
+              service_type: {
+                type: "string",
+                enum: [
+                  "blog",
+                  "email",
+                  "facebook",
+                  "flickr",
+                  "foursquare",
+                  "google-plus",
+                  "instagram",
+                  "linkedin",
+                  "other",
+                  "pinterest",
+                  "twitter",
+                  "youtube",
+                ],
+              },
+              title: {
+                type: "string",
+              },
+              href: {
+                type: "string",
+                format: "uri",
+              },
+              image: {
+                "$ref": "#/definitions/image",
+              },
+            },
+          },
+          description: "A set of links to social media profiles for the organisation.",
+        },
+        external_related_links: {
+          "$ref": "#/definitions/external_related_links",
+        },
+        tags: {
```

Comment:
What are `tags` for?

Reply:
Hangover from copying the placeholder schema - removed